packages/fluent-theme/src/fluent/styles/ChoiceGroupOption.styles.ts

```diff
@@ -3,6 +3,8 @@ import { IChoiceGroupOptionStyleProps, IChoiceGroupOptionStyles } from 'office-u
 export const ChoiceGroupOptionStyles = (props: IChoiceGroupOptionStyleProps): Partial<IChoiceGroupOptionStyles> => {
   const { checked, disabled, theme, hasIcon, hasImage } = props;
   const { semanticColors, palette } = theme;
+  const labelWrapperLineHeight = 15;
```

Comment:
This feels so random... and later you are not even using it directly but do a math on it.